app/controllers/contract_controller.rb

```diff
@@ -0,0 +1,4 @@
+class ContractController < ApplicationController
+  def index
```

Comment:
where are the other operations? edit, new.. post.. delete...?

Reply:
The best thing you can do is to remove the file and run this command: `rails g controller ContractsController`